charts/media-common/Chart.yaml

```diff
@@ -2,10 +2,16 @@ apiVersion: v2
 name: media-common
 description: Common dependancy chart for media ecosystem containers
 type: application
-version: 1.0.1
+version: 1.1.0
```

Comment:
Technically this is correct in semver bumping. Doing this _will_ cause the already created containers to desync. 
I think we need to bump the semver regex to be a minor match instead of a micro match.
E.g. `~1.0.0` to `^1.0.0`

Reply:
Will update the dependent charts in another PR 👍